public/BUILD.bazel

```diff
@@ -2,9 +2,5 @@ package(default_visibility = ["//visibility:public"])
 
 filegroup(
     name = "public",
-    srcs = glob(["assets/**/*"]) + glob(["build/**/*"]) + [
-        "favicon.png",
-        "global.css",
-        "index.html",
-    ],
-)
+    srcs = glob(["assets/**/*"]) + glob(["build/**/*"])
+)
```

Comment:
can you please run `make buildifier` to clean up the build files.

Reply:
done in a6606a5